src/Core/Domain/AttributeGroup/ValueObject/AttributeGroupId.php

```diff
@@ -65,7 +65,7 @@ public function getValue(): int
      */
     private function assertIntegerIsGreaterThanZero(int $attributeGroupId): void
     {
-        if (0 >= $attributeGroupId) {
+        if (0 > $attributeGroupId) {
```

Comment:
The VO for IDs must validate that the value is valid, so strictly positive This shouldn't be modified

Why did you need to change this? Is it because of the form type that fetches the parent AttributeGroup? If so then it should be check in the form type itself not to fetch anything when `$attributeGoupId` is 0 (which it actually seems to be doing already)

Reply:
Because it’s called « is greater » and not « isGreaterOrEqual ».